src/rgw/rgw_sal_motr.cc

```diff
@@ -1294,51 +1294,61 @@ MotrObject::~MotrObject() {
 //    return read_op.prepare(dpp);
 //  }
 
-int MotrObject::set_obj_attrs(const DoutPrefixProvider* dpp, RGWObjectCtx* rctx, Attrs* setattrs, Attrs* delattrs, optional_yield y, rgw_obj* target_obj)
+int MotrObject::fetch_obj_entry_and_key(const DoutPrefixProvider* dpp, rgw_bucket_dir_entry& ent, std::string& bname, std::string& key, rgw_obj* target_obj)
 {
-  // TODO : Set tags for multipart objects
-  if (this->category == RGWObjCategory::MultiMeta)
-    return 0;
+    int rc = this->get_bucket_dir_ent(dpp, ent);
+  if (rc < 0) {
```

Comment:
Please check the alignment.
Also, you can add a comment (line no 1301) like below:
`ldpp_dout(dpp, 0) <<__func__<< " : ERROR: failed to get object entry. rc=" << rc << dendl;`

Reply:
done